data/coalition/coalition missions.txt

```diff
@@ -1453,7 +1453,7 @@ mission "Coalition: Alpha Encounter"
 			choice
 				`	"Are you so busy in that empty cell that you can't answer me?"`
 				`	"Tough talk for someone who wrestled a spider and lost."`
-			`	He leans forward, his legs twitching as if stopped mid-tackle, and he pauses, turning to look down the hallway to where the Heliarchs went. He backs up, slowing his breathing, and then sits down. "Alright. This may be the last time in three years I can have a proper conversation, so I will entertain you. My name... is Alitis. Over a century ago my kin stumbled upon an alien civilization beyond the Far North in conflict with bands of invaders. These invaders possessed drive capable of jumping between systems without the need for hyperspace links, technology that we greatly desired, for it could put us far outside of the reach of... you." He fixates his eyes on yours, ever so slightly closing his eyelids. "We explored the far reaches of the galaxy to the east, but found no place to call home. Our only choice was to return to the Dirt Belt where we knew of an enclave of more kin, only for the Navy," he spits on the ground, then looks at you again, "to find us. We evacuated, and my kin thought it wise to return to the east, but I knew better. I stole one of the drives my kin had along with a small ship and began traveling west. That is when I discovered the Coalition, and they have provided for me a home that I have been unable to find elsewhere in the galaxy."`
+			`	He leans forward, his legs twitching as if stopped mid-tackle, and he pauses, turning to look down the hallway to where the Heliarchs went. He backs up, slowing his breathing, and then sits down. "Alright. This may be the last time in three years I can have a proper conversation, so I will entertain you. My name... is Alitis. Over a century ago my kin stumbled upon an alien civilization beyond the Far North in conflict with bands of invaders. These invaders possessed devices capable of jumping between systems without the need for hyperspace links, technology that we greatly desired, for it could put us far outside of the reach of... you." He fixates his eyes on yours, ever so slightly closing his eyelids. "We explored the far reaches of the galaxy to the east, but found no place to call home. Our only choice was to return to the Dirt Belt where we knew of an enclave of more kin, only for the Navy," he spits on the ground, then looks at you again, "to find us. We evacuated, and my kin thought it wise to return to the east, but I knew better. I stole one of the drives my kin had along with a small ship and began traveling west. That is when I discovered the Coalition, and they have provided for me a home that I have been unable to find elsewhere in the galaxy."`
```

Comment:
I'm pretty sure the intention was "drives" plural rather than "drive" singular or "devices," as latter on in this paragraph it reads "I stole one of the drives..." It's referring to hyperdrives/scram drives/jump drives generally as "drives."

Reply:
Either or, really. I think it's fine to refer to the jump drive as a device.

Reply:
Personally, both sounds alright. Would prefer it to be in plural, though.

Reply:
The main reason I go for "devices" than "drives" is because I'm not sure "drives" has been coined as a general noun referring to [Hyperdrive, Scram Drive, Jump Drive]. The three (and more), however, are definitely devices.

I can also change the "I stole one of the drive" to "I stole one of the devices" later too, if that's a concern. Or change both to "drives". All are viable alternatives.

Reply:
I am able to find a few other instances of "drive" being used to refer to FTL devices.
https://github.com/endless-sky/endless-sky/blob/58f23f273aa4bf2d5fdccb8ed28b32ebcac395c4/data/kahet/kahet%20missions.txt#L428
https://github.com/endless-sky/endless-sky/blob/58f23f273aa4bf2d5fdccb8ed28b32ebcac395c4/data/coalition/coalition%20jobs.txt#L1347

Reply:
So, should I change it to "drives"?

Reply:
Yes.